src/Ng/Jobs/EndpointMonitoringJob.cs

```diff
@@ -0,0 +1,133 @@
+                   + $"-{Arguments.StorageBaseAddress} <storage-base-address> "
+                   + $"-{Arguments.StorageType} azure "
+                   + $"[-{Arguments.StoragePath} <path>]"
+                   + "|"
+                   + $"[-{Arguments.StorageAccountName} <azure-acc> "
+                   + $"-{Arguments.StorageKeyValue} <azure-key> "
+                   + $"-{Arguments.StorageContainer} <azure-container> "
+                   + $"-{Arguments.StoragePath} <path> "
+                   + $"[-{Arguments.VaultName} <keyvault-name> "
+                   + $"-{Arguments.ClientId} <keyvault-client-id> "
+                   + $"-{Arguments.CertificateThumbprint} <keyvault-certificate-thumbprint> "
+                   + $"[-{Arguments.ValidateCertificate} true|false]]] "
+                   + $"[-{Arguments.StoragePathAuditing} <path>]"
+                   + "|"
+                   + $"[-{Arguments.StorageAccountNameAuditing} <azure-acc> "
+                   + $"-{Arguments.StorageKeyValueAuditing} <azure-key> "
+                   + $"-{Arguments.StorageContainerAuditing} <azure-container> "
+                   + $"-{Arguments.StoragePathAuditing} <path>] "
+                   + $"[-{Arguments.Verbose} true|false] "
+                   + $"[-{Arguments.Interval} <seconds>]";
+        }
+
+        private IList<Lazy<INuGetResourceProvider>> ResourceProvidersToInjectV2 => new List<Lazy<INuGetResourceProvider>>
+        {
+            new Lazy<INuGetResourceProvider>(() => new NonhijackableV2HttpHandlerResourceProvider()),
+            new Lazy<INuGetResourceProvider>(() => new PackageTimestampMetadataResourceV2Provider(LoggerFactory))
+        };
+
+        private IList<Lazy<INuGetResourceProvider>> ResourceProvidersToInjectV3 => new List<Lazy<INuGetResourceProvider>>
+        {
+        };
+
+        protected override void Init(IDictionary<string, string> arguments, CancellationToken cancellationToken)
+        {
+            var gallery = arguments.GetOrThrow<string>(Arguments.Gallery);
+            var index = arguments.GetOrThrow<string>(Arguments.Index);
+            var source = arguments.GetOrThrow<string>(Arguments.Source);
+            var verbose = arguments.GetOrDefault(Arguments.Verbose, false);
+
+            if (arguments.GetOrThrow<string>(Arguments.StorageType) != Arguments.AzureStorageType)
+            {
+                throw new ArgumentException("File storage is not supported!");
+            }
+
+            var storageFactory = CommandHelpers.CreateStorageFactory(arguments, verbose);
+            var auditingStorageFactory = CommandHelpers.CreateSuffixedStorageFactory("Auditing", arguments, verbose);
+
+            var endpointNames = arguments.GetOrThrow<string>(Arguments.EndpointsToTest).Split(';');
+
+            Logger.LogInformation(
+                "CONFIG gallery\"{Gallery}\" source: \"{ConfigSource}\" storage: \"{Storage}\" auditingStorage: \"{AuditingStorage}\" endpoints: \"{Endpoints}\"",
```

Comment:
Serilog wraps the values with quotes [automagically](https://github.com/serilog/serilog/wiki/Writing-Log-Events).